### DISCUSSION TITLE
[bug report] can't select type of feedback in review

### DISCUSSION THREAD

**Discussion by asg0451** [community/community#6840] [Pull Requests] [answered]
![image](https://user-images.githubusercontent.com/4358545/139127326-ccde2c4d-6032-4269-bbe5-b1f55803e2fc.png)
windows 10, recent chrome, dark mode

**Comment by julik** [comment]
Also noted here https://github.com/github/feedback/discussions/6852

**Comment by gaknoll** [comment, marked as answer]
Thank you @asg0451 for reporting this, and @julik for cross-posting! We are looking into this and hope to get a fix out soon.

**Reply by gaknoll** [reply]
@asg0451 The team dug into this, and discovered it to be caused by the "Save to Pocket" Chrome extension. Pocket is aware and we hope they get this resolved quickly!
https://twitter.com/Phacks/status/1453343546371674118

**Reply by asg0451** [reply]
awesome, thanks! I do use that extension yeah

**Reply by julik** [reply]
Thank you 👍  let's hope they resolve it soon.
